app/src/main/java/org/mozilla/tv/firefox/navigationoverlay/ToolbarUiController.kt

```diff
@@ -58,7 +57,7 @@ class ToolbarUiController(
             }
         }
 
-        val layoutInflater = layout.context.systemService<LayoutInflater>(Context.LAYOUT_INFLATER_SERVICE)
+        val layoutInflater = layout.context.getSystemService<LayoutInflater>()!!
```

Comment:
nit: we could get rid of the `!!` if we use `LayoutInflater.from(layout.context)`